src/run.rs

```diff
@@ -27,23 +36,25 @@ where
     I: IntoIterator<Item = T>,
     T: Into<OsString> + Clone,
 {
-    let mut args = args.into_iter().map(|s| s.into()).collect::<Vec<_>>();
-    let cli = Cli::parse_from(args.clone());
-    let name = args.remove(0);
-    if let Some(venv_aqora) = find_venv_aqora(name, &cli) {
-        args.push("--ignore-venv-aqora".into());
-        let status = std::process::Command::new(venv_aqora)
-            .args(args)
-            .status()
-            .unwrap();
-        return status.code().unwrap_or(1).try_into().unwrap_or(1);
-    }
-    let tokio = TOKIO.get_or_init(|| {
-        tokio::runtime::Builder::new_multi_thread()
-            .enable_all()
-            .build()
-            .unwrap()
-    });
+    let cli = if cfg!(debug_assertions) {
```

Comment:
I'm not sure I fully understand why this is necessary? why would we want debug behavior to differ from release behavior?

Reply:
This change forces the CLI to execute its own code instead of delegating to the one in the use-case/submission venv, which usually happens to be a release binary.